pkl-core/src/main/java/org/pkl/core/Evaluator.java

```diff
@@ -70,6 +80,16 @@ static Evaluator preconfigured() {
    */
   Object evaluateOutputValue(ModuleSource moduleSource);
 
+  /**
+   * Evaluates a module's {@code output.value} property, returning a byte array of the <a
+   * href="https://pkl-lang.org/main/current/bindings-specification/binary-encoding.html"><code>
+   * pkl-binary</code></a>-encoded representation of the result.
+   *
+   * @throws PklException if an error occurs during evaluation
+   * @throws IllegalStateException if this evaluator has already been closed
+   */
+  byte[] evaluateOutputValuePklBinary(ModuleSource moduleSource);
+
```

Comment:
I think let's just keep `evaluateExpressionPklBinary` and eliminate `evaluatePklBinary` and `evaluateOutputValuePklBinary`.

This is an intentionally low-level way to eval, and I think it's okay that we have just one method.
Users that are interested in evaluating the whole module can do `evaluateExpressionPklBinary(myModule, "module")`;.